code/src/App.js

```diff
@@ -1,9 +1,36 @@
-import React from 'react'
+import React, { useState } from "react";
+import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
+
+import { Header } from "./components/Header";
+import { MovieList } from "./pages/MovieList";
+import { MovieDetails } from "./pages/MovieDetails";
+import { NotFound } from "./pages/NotFound";
+import { POPULAR_URL } from "./urls";
 
 export const App = () => {
+  const [url, setUrl] = useState(POPULAR_URL);
+
+  const getUrl = (url) => {
+    setUrl(url);
+  };
```

Comment:
So, get or set? :D It's crucial difference in meaning. Get is when we query for some data. Set is when we setting data somewhere.
I would suggest you to name your function like this :

```
  const setCurrentUrl = (url) => {
    setUrl(url);
  };
```

Reply:
Oh wow, I can't really explain this 😬
Thank you for pointing it out, I will change this.